src/Components/RecommendationsPage/RecommendationsTable.tsx

```diff
@@ -136,14 +176,53 @@ const RecommendationsTable = ({ page, perPage, setTotal }) => {
                     {rec.remediation === 'Playbook' ? <PlayIcon /> : <WrenchIcon />} {rec.remediation}
                   </Td>
                   <Td dataLabel={columnNames.run} modifier="fitContent">
-                    {rec.remediation === 'Playbook' && <Button variant="primary">{columnNames.run}</Button>}
+                    {rec.remediation === 'Playbook' && (
+                      <Button
+                        variant="primary"
+                        onClick={() => {
+                          setCurrentRule(rec);
+                          getResolutionsInfo(rec);
+                          setModalIsOpen(true);
+                        }}
+                      >
+                        {columnNames.run}
+                      </Button>
+                    )}
                   </Td>
                 </Tr>
               ))
             )}
           </Tbody>
         )}
       </Table>
+
+      {modalIsOpen && (
+        <SystemPickerModal
+          isOpen={modalIsOpen}
+          setIsOpen={setModalIsOpen}
+          title={`Create remediation playbook for ${ruleName}`}
+          onSubmit={createRemediation}
+          submitText={'Create remediation playbook'}
+          header={
+            <>
+              <Flex style={{ paddingBottom: '8px' }}>
+                <FlexItem style={{ width: '100%' }}>
+                  <TextContent>
+                    <Text component={TextVariants.p}>{currentRule?.description}</Text>
+                  </TextContent>
+                  <br />
+                  <TextContent>
+                    <Text component={TextVariants.p}>
+                      The playbook <b>{ruleName}</b> does <span style={{ color: 'red' }}>{currentResolutions?.needs_reboot ? '' : 'not'}</span> auto
+                      reboot systems.
+                    </Text>
+                  </TextContent>
```

Comment:
TO be consistent with tasks, it would be nice to also have the header with title saying  'select systems to execute', an info alert about connection, for remediation it has to be connected in same way

![image](https://github.com/RedHatInsights/in-place-upgrades-frontend/assets/19702477/1bb29d6b-41f2-4865-8f95-cb292eda4002)